modules/nw-dual-stack-convert.adoc

```diff
@@ -33,36 +27,59 @@ After converting to dual-stack networking only newly created pods are assigned I
   path: /spec/serviceNetwork/-
   value: fd02::/112 <2>
 ----
-<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 CIDR prefix must be large enough to accommodate the specified host prefix.
-
+<1> Specify an object with the `cidr` and `hostPrefix` fields. The host prefix must be `64` or greater. The IPv6 Classless Inter-Domain Routing (CIDR) prefix must be large enough to accommodate the specified host prefix.
 <2> Specify an IPv6 CIDR with a prefix of `112`. Kubernetes uses only the lowest 16 bits. For a prefix of `112`, IP addresses are assigned from `112` to `128` bits.
---
 
-. To patch the cluster network configuration, enter the following command:
+. To patch the cluster network configuration, enter the following command in your CLI:
```

Comment:
"cluster network configuration" reads a bit too backend. Would "cluster's network " work?

Reply:
How about "To patch the configuration"? It should be clear, if you went so far in this doc, that we are talking about network confguration